CRITICAL - 2023-10-15 03:59:22 --> Undefined variable $data
in APPPATH\Controllers\IndexController.php on line 73.
 1 APPPATH\Controllers\IndexController.php(73): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $data', 'C:\\xampp\\htdocs\\projectfyp\\app\\Controllers\\IndexController.php', 73)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 04:00:25 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projectfyp\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(60): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#11 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-10-15 04:00:25 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\IndexController.php(60): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 04:01:40 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projectfyp\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(60): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#11 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-10-15 04:01:40 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\IndexController.php(60): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 04:04:23 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projectfyp\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(60): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#11 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-10-15 04:04:23 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\IndexController.php(60): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 04:17:57 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projectfyp\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(61): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#11 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-10-15 04:17:57 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\IndexController.php(61): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 04:23:36 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projectfyp\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(65): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#11 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-10-15 04:23:36 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `password`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :password:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\IndexController.php(65): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 04:35:36 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(65): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 04:35:36 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `password`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :password:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(65): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-15 04:37:56 --> Call to undefined method App\Models\UserModel::addUser
in SYSTEMPATH\Model.php on line 873.
 1 APPPATH\Controllers\IndexController.php(65): CodeIgniter\Model->__call('addUser', [...])
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 04:39:24 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(65): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 04:39:24 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `password`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :password:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(65): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-15 04:47:40 --> Undefined variable $model
in APPPATH\Controllers\IndexController.php on line 49.
 1 APPPATH\Controllers\IndexController.php(49): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $model', 'C:\\xampp\\htdocs\\projectfyp\\app\\Controllers\\IndexController.php', 49)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-15 04:48:17 --> Undefined variable $model
in APPPATH\Controllers\IndexController.php on line 64.
 1 APPPATH\Controllers\IndexController.php(64): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $model', 'C:\\xampp\\htdocs\\projectfyp\\app\\Controllers\\IndexController.php', 64)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 04:48:55 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(64): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 04:48:55 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(64): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 05:03:02 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(64): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 05:03:02 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `email`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :email:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(64): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 05:04:31 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(64): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 05:04:31 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `email`, `password`, `address`, `register_date`, `level_id`, `updated_at`) VALUES (:username:, :full_name:, :email:, :password:, :address:, :register_date:, :level_id:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(64): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 05:08:32 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 05:08:32 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `email`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :email:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 05:12:22 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(64): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 05:12:22 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `email`, `password`, `address`, `register_date`, `level_id`, `updated_at`) VALUES (:username:, :full_name:, :email:, :password:, :address:, :register_date:, :level_id:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(64): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 05:14:40 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(64): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 05:14:40 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `email`, `password`, `address`, `register_date`, `level_id`, `updated_at`) VALUES (:username:, :full_name:, :email:, :password:, :address:, :register_date:, :level_id:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(64): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 05:15:18 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(64): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 05:15:18 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `email`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :email:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(64): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 05:16:54 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(64): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 05:16:54 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `email`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :email:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(64): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 05:17:12 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(64): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 05:17:12 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `email`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :email:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(64): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 05:20:33 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 05:20:33 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `email`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :email:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 05:36:09 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(64): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 05:36:09 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `email`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :email:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(64): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-15 05:37:55 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projectfyp\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\projectfyp\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\projectfyp\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\projectfyp\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projectfyp\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projectfyp\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projectfyp\app\Controllers\IndexController.php(64): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
#9 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
#10 C:\xampp\htdocs\projectfyp\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projectfyp\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-15 05:37:55 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `full_name`, `email`, `password`, `address`, `level_id`, `register_date`, `updated_at`) VALUES (:username:, :full_name:, :email:, :password:, :address:, :level_id:, :register_date:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\IndexController.php(64): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-15 06:12:28 --> Undefined variable $alertBody
in APPPATH\Views\alertsuccess.php on line 8.
 1 APPPATH\Views\alertsuccess.php(8): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $alertBody', 'C:\\xampp\\htdocs\\projectfyp\\app\\Views\\alertsuccess.php', 8)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\projectfyp\\app\\Views\\alertsuccess.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('alertsuccess', [], true)
 5 APPPATH\Controllers\IndexController.php(67): view('alertsuccess', [...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\IndexController->signupProcess()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\IndexController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
